src/plugins/data_views/server/routes/fields_for.ts

```diff
@@ -91,6 +92,10 @@ const handler: RequestHandler<{}, IQuery, IBody> = async (context, request, resp
       fields: request.query.fields,
     });
 
+    if (indices.length === 0 && allowNoIndex !== true) {
+      throw new DataViewMissingIndices(pattern);
+    }
```

Comment:
Why is this needed? shouldn't it just return an empty array of fields + indices in this case?

Reply:
@mattkime these changes should be removed, to make the PR green